project/Dependencies.scala

```diff
@@ -148,6 +148,8 @@ object Dependencies {
     logbackDep,
     "io.monix"   %% "monix-kafka-1x" % monixKafkaVersion,
     "org.rogach" %% "scallop"        % "3.1.1",
+    "com.google.protobuf" % "protobuf-java" % "3.21.7",
```

Comment:
I think we have proto and snappy dependencies defined for projects as well. Should we make sure that we are using the same versions across ( to avoid runtime overrides ) ?

<img width="691" height="505" alt="image" src="https://github.com/user-attachments/assets/18a7cbe6-5dab-4b7c-8896-e73feb0f61da" />